storm-core/src/ui/public/index.html

```diff
@@ -143,6 +143,12 @@
             ]
           });
           $('#topology-summary [data-toggle="tooltip"]').tooltip();
+          $.getJSON("/api/v1/cluster/configuration", function(json){
```

Comment:
i suggest we avoid making another ajax call - instead we can have topology summary include boolean as part of topology-summary.